concept/ConceptManager.java

```diff
@@ -325,6 +329,23 @@ private BiFunction<ConceptManager, TypeVertex, AttributeType> attributeTypeConst
         throw exception(TypeDBException.of(BAD_VALUE_TYPE, valueType));
     }
 
+    public void validateNotRoleTypeAlias(Label label) {
```

Comment:
We put role type alias validation in the `ConceptManager` to centralise the semantics in one place